runtime/vm/lookupmethod.c

```diff
@@ -379,7 +379,9 @@ javaResolveInterfaceMethods(J9VMThread *currentThread, J9Class *targetClass, J9R
 			BOOLEAN shouldAdd = TRUE;
 			J9Class* newMethodClass = NULL;
 			newMethod = processMethod(currentThread, lookupOptions, foundMethod, interfaceClass, &data->exception, &data->exceptionClass, &data->errorType, nameAndSig, senderClass, targetClass);
-			if (NULL == newMethod) {
+			if (NULL != currentThread->currentException) {
+				return NULL;
```

Comment:
Can this one be a goto like the others rather than an inline return?

Reply:
this function uses inline return consistently, would you like me to change all usage to `goto end`?